avsomr/CMakeLists.txt

```diff
@@ -72,13 +72,13 @@ if (NOT MSVC)
    set_target_properties (
       avsomr
       PROPERTIES
-         COMPILE_FLAGS "${PCH_INCLUDE} -g -Wall -Wextra -Winvalid-pch -Wno-unused-private-field"
```

Comment:
no such option in MinGW 7.30. And no such warning either

Reply:
Or is this Clang only maybe? @igorkorsukov?

Reply:
To make the project work like the others, I just copied it from https://github.com/musescore/MuseScore/blob/master/omr/CMakeLists.txt  :)

Reply:
I see, that code hasn't been used since quite long. So my fix here should be ok